src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
+$duration: 300ms;
```

Comment:
It is better to define the variable $duration inside the corresponding SCSS files (variables.scss)